app/workers/classified_listings/bust_cache_worker.rb

```diff
@@ -0,0 +1,14 @@
+module ClassifiedListings
+  class BustCacheWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(classified_listing_id, cache_buster = "CacheBuster")
```

Comment:
I think we can get rid of the `cache_buster` argument here and call `CacheBuster.bust_classified_listings` directly

Reply:
ok, I was unsure about removing it because I do not have the big picture :) but I will do it :)